apps/devhub/views.py

```diff
@@ -764,7 +764,7 @@ def json_file_validation(request, addon_id, addon, file_id):
             return http.HttpResponseNotAllowed(['POST'])
 
         try:
-            v_result = tasks.file_validator(file.id)
+            v_result = tasks.validate(file).get()
```

Comment:
It causes us to wait for the validation task to complete, and return the result. I'll add a comment.